src/DataShare/DataShare/Account/NewAzDataShareAccount.cs

```diff
@@ -74,22 +73,15 @@ public class NewAzDataShareAccount : AzureDataShareCmdletBase
 
         public override void ExecuteCmdlet()
         {
-            this.ConfirmAction(
-                string.Format(Resources.ResourceCreateConfirmation, this.Name),
-                this.Name,
-                this.NewAccount);
-        }
-
-        private void NewAccount()
-        {
-            if (this.ShouldProcess(this.Name, "Create"))
+            if (this.ShouldProcess(this.Name, $"Creating data share account '{this.Name}'."))
```

Comment:
In the second parameter "this.name" is not needed. Should process constructs the message with both the parameters.

Reply:
their design documentation mentions it this way and he asked me to follow the design documentation